app/views/hub/zip_codes/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<h2>Zip Code Routing</h2>
+<% if @success_message %>
+  <p aria-live="assertive" class="sr-only"><%= @success_message %></p>
+<% end %>
+
+<ul class="organization-list">
+  <% form.vita_partner.serviced_zip_codes.each do |szc| %>
+  <% next unless szc.id.present? %>
```

Comment:
Would this case ever occur? I can't see a reason it would, so I encourage you to delete this line.